src/drive/web/modules/filelist/FileThumbnail.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { withBreakpoints } from 'cozy-ui/react'
+import { ImageLoader } from 'components/Image'
+import { SharedBadge } from 'sharing'
+import FileIcon from './FileIcon'
+import styles from 'drive/styles/filelist.styl'
+
+const FileThumbnail = ({
+  file,
+  withSharedBadge,
+  breakpoints: { isMobile }
+}) => {
+  const isImage = file.class === 'image'
+  return (
+    <div
+      className={cx(styles['fil-content-cell'], styles['fil-file-thumbnail'], {
+        'u-pl-0': !isMobile
```

Comment:
can't we use a css cozy-ui breakpoint here?

Reply:
What do you mean? `isMobile` comes from cozy-ui and is based on the same breakpoints.

Reply:
We don't have new utilities classes to manage this kind of thing? u-pl-0-XXX ?

Reply:
The margin/padding classes are not responsive no. There aren't many cases where we ned this, AFAIK.